diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter2/comments/CommentsRule.kt

```diff
@@ -70,8 +73,19 @@ class CommentsRule(private val configRules: List<RulesConfig>) : Rule("comments"
                         offset to ktPsiFactory.createImportDirective(ImportPath.fromString(text.substringAfter("$importKeyword "))).node
                     text.contains(packageKeyword) ->
                         offset to ktPsiFactory.createPackageDirective(FqName(text.substringAfter("$packageKeyword "))).node
-                    else ->
-                        offset to ktPsiFactory.createBlockCodeFragment(text, null).node
+                    else -> {
+                        var newNode = ktPsiFactory.createBlockCodeFragment(text, null).node
+                        val isPossibleError = newNode.findChildByType(BLOCK)!!.firstChildNode.elementType == BINARY_EXPRESSION
```

Comment:
Is `!!` here because `createBlockCodeFragment` should always create block?

Reply:
Yes, it is